apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/UpdateSearchRequest.tsx

```diff
@@ -134,37 +134,47 @@ export const UpdateSearchRequestForm = ({
           <form onSubmit={handleSubmit(handleSaveNotes)}>
             <Heading level="h2" size="h4">
               {intl.formatMessage({
-                defaultMessage: "Personal Notes",
-                id: "l05aVF",
+                defaultMessage: "Notes about this request",
+                id: "peTSHR",
                 description:
-                  "Heading for the personal notes section of the single search request view.",
+                  "Heading for the notes about this request section of the single search request view.",
               })}
             </Heading>
+            <p data-h2-padding="base(x.5, 0, x.5, 0)">
+              {intl.formatMessage({
+                defaultMessage:
+                  "These notes are available to all managers of this pool, but not to candidates.",
+                id: "VkBcGJ",
+                description:
+                  "Description for the notes about this request section of the single search request view.",
+              })}
+            </p>
             <TextArea
               id="adminNotes"
               name="adminNotes"
               label={intl.formatMessage({
-                defaultMessage: "Personal Notes",
-                id: "p7D5i5",
+                defaultMessage: "Request notes",
+                id: "Pe1kas",
                 description:
-                  "Label displayed on the search request form personal notes field.",
+                  "Label displayed on the search request form request notes field.",
               })}
               rows={8}
+              required={false}
```

Comment:
I couldn't find anything in the issue related to this, was there a reason for adding `required={false}`?

Reply:
This is Optional field

Reply:
Jerbo had that label in figma .. since we longer show that label on ui I asked him to remove it but kept the filed as optional

Reply:
It seems unintuitive but, in order to set an HTML boolean value, you simply omit it.[ Passing `false` to the `required` attribute is technically not valid HTML](https://www.w3.org/TR/2011/WD-html5-author-20110809/common-microsyntaxes.html#boolean-attribute:~:text=The%20values%20%22true%22%20and%20%22false%22%20are%20not%20allowed%20on%20boolean%20attributes.%20To%20represent%20a%20false%20value%2C%20the%20attribute%20has%20to%20be%20omitted%20altogether.) and is ignored.

Reply:
removed in [7d0be14](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7298/commits/7d0be145bb8c1be2f30cccf9d0dee6468d496a18)